### DISCUSSION TITLE
voxelization_toolkit

### DISCUSSION THREAD

**Discussion by DaivaMarcinkeviciute** [IfcOpenShell/IfcOpenShell#4246] [Q&A]
I have installed ifcopenshell and voxelization_toolkit ( https://github.com/opensourceBIM/voxelization_toolkit/blob/3d15982b7d698c2d033df13f0a0724eb9bfb8358/python/voxec/__init__.py#L214 ) with anaconda using python 3.11.
However i cannot find any information on how to import the tools from the voxelization_toolkit.
i tried following and got en error message:
`import voxelization_toolkit`
 and
`from ifcopenshell import voxelization_toolkit`   

Could anyone help me with that? Thank you.

**Comment by aothms** [comment]
This is a bit confusing still. We're working on an actual python wrapper that you can import, but for the time being you have to use the executable. The executable (`voxec` or `voxec.exe`) is installed by using anaconda and then you can copy over and adapt these files manually to interact with the exe https://github.com/opensourceBIM/voxelization_toolkit/tree/master/python/voxec

**Comment by Moult** [comment]
By the way, there is a GSoC project about better integration with the voxelisation toolkit if you're interested in participating :) https://github.com/opencax/GSoC/issues/88
